sdk/aqueduct/operators.py

```diff
@@ -151,6 +155,7 @@ class OperatorSpec(BaseModel):
     metric: Optional[MetricSpec]
     check: Optional[CheckSpec]
     param: Optional[ParamSpec]
+    systemmetric: Optional[SystemMetricSpec]
```

Comment:
some comment as in the backend PR about using underscores to delimit multiple words